src/app/system/devTools/DevMenu/DevMenu.tsx

```diff
@@ -648,3 +674,39 @@ export const DevMenuButtonItem: React.FC<{
     </Touchable>
   )
 }
+
+const ExperimentFlagItem: React.FC<{ description: string; flag: string }> = ({
+  description,
+  flag,
+}) => {
+  const experiment = useExperimentVariant(flag)
+  return (
+    <Flex py={1} px={2}>
+      <Text fontWeight="bold" selectable>
+        {flag}
+      </Text>
+      <Text>
+        Status:{" "}
+        {experiment.enabled ? (
+          <Text fontWeight="bold" color="blue100">
+            enabled
+          </Text>
+        ) : (
+          <Text fontWeight="bold" color="red100">
+            disabled
+          </Text>
+        )}
+      </Text>
+      <Text>
+        Variant: <Text fontWeight="bold">{experiment.variant}</Text>
+      </Text>
+      <Flex flexDirection="row" flexWrap="wrap">
+        <Text>Payload: </Text>
+        <Text fontWeight="bold" selectable>
+          {experiment.payload}
+        </Text>
+      </Flex>
+      <Text>Description: {description}</Text>
```

Comment:
Could be helpful to be able to copy the Unleash URL.

```suggestion
      <Text>Description: {description}</Text>
      <Text selectable>{`https://unleash.artsy.net/projects/default/features/${flag}`}</Text>
```

Reply:
great idea! will do that

Reply:
done - but since the url is long i changed the approach as in here

https://github.com/artsy/eigen/assets/11945712/a7b33bbd-3023-4257-a191-072ee72c0536